e2e_test/source_inline/cdc/mysql/mysql_create_drop.slt.serial

```diff
@@ -45,7 +45,6 @@ create source s with (
   username = 'shared-cdc',
   password = 'abcdef',
   database.name = 'testdb1',
-  server.id = '114514'
```

Comment:
Manually tested that before this PR, this test will fail if `server.id` not specified